composer.json

```diff
@@ -17,10 +17,11 @@
     }
   ],
   "require": {
-    "php": ">=8.0",
+    "php": ">=8.1",
     "ext-dom": "*",
     "ext-libxml": "*",
     "php-webdriver/webdriver": "^1.8.2",
+    "phpstan/phpstan": "^1.10",
```

Comment:
this should be removed (PHPStan is not a hard dependency of this project).

Reply:
moved to dev dependencies: https://github.com/arderyp/panther/commit/5cf344a56fbadfced1d5f538b20a805803117b3f

Reply:
or maybe you don't even want it in the dev dependencies? If so, I can delete this and `php-cs-fixer` from dev dependencies, just let me know.

Reply:
Yes please delete it. The bot uses PHP CS Fixer under the hood but we don't need to install it locally.

Reply:
I've deleted phpstan and php-cs-fixer from dev dependencies: https://github.com/arderyp/panther/commit/cdcd7e08b58d93bf10a74c67cc6dbbde097dee89

That said, this is a confusing decision to me. Presumably, we should all be running phpstan before committing and only commit non-failing code. If so, why not keep phpstan as a dev dependency? Otherwise devs need to know to manually install it, then run it, then remember to manually delete it, then do it all over again for each commit. Furthermore, having it as a dev dependency doesn't hurt anything (or maybe I'm wrong?).

Likewise, on my initial PR, one of the Github actions failed for fixer related issues. If contributors are supposed to run the fixer themselves manually before committing, then it's the same weird expectation on dev to know they need to install it manually, run it, then remove it before commit, then do it all over again for the next commit.

@dunglas resolve if we don't need to run fixer/phpstan manually before committing (or we do and you still think they shouldn't be a dev dependencies for some reason)